composer.json

```
@@ -185,6 +185,7 @@
         "nikic/php-parser":"^4.7",
         "doctrine/inflector": "^1.4",
         "sylius/grid-bundle": "1.7.4",
+        "symfony/doctrine-bridge": "4.4.15",
```

Shouldn't it be a conflict with this version?:
```suggestion
        "symfony/doctrine-bridge": "4.4.16",
```